app/javascript/components/widgets/messages/default-messages.jsx

```diff
@@ -0,0 +1,2 @@
+export const LoadingMessage = () => 'Loading...';
```

Comment:
I'm still semi-new to React myself, but it strikes me as odd that we'd want to use a Component to return a simple string here.  I think this would be better handled with just straight JS.

Reply:
@thedelchop From a consistency perspective, I like using `<LoadingMessage />` instead of needing to interpolate (`{LoadingMessage}`). Another advantage is that it hides the details of implementation from the consumer side, allowing us to update `LoadingMessage` later to be more robust without modifying the call sites (e.g., `export const LoadingMessage = () => <BigLetters>Loading</BigLetters>`)